shared/templates/static/ansible/ensure_gpgcheck_local_packages.yml

```diff
@@ -0,0 +1,38 @@
+# platform = multi_platform_rhel, multi_platform_fedora
+# reboot = false
+# strategy = unknown
+# complexity = low
+# disruption = medium
+- name: "Check existence of yum on Fedora"
+  stat:
+    path: /etc/yum.conf
+  register: yum_config_file
```

Comment:
Won't this be in the playbook multiple times because we have it in multiple fixes?

Reply:
Every time this register occurs, it will override the previous register. So for that, this should only be used for this task, assuming people are registering the same variable in a later play.